static/css/rue89.css

```diff
@@ -0,0 +1,134 @@
+a:link,
```

Comment:
Why having included the rules as CSS? Using SASS or LESS would give more flexibility, allowed to integrate with the theme/platform variables & mixins, and made it less verbose.